src/ensembl/src/shared/components/textarea/Textarea.scss

```diff
@@ -0,0 +1,17 @@
+.textarea {
+  display: inline-block;
+  width: 100%;
+  border: none;
+  padding: 7px;
+  font-size: 16px;
```

Comment:
Default font-size of an input is 14px. Should default font-size of textarea be larger?

Reply:
No, I've removed it.